test/perf/tests/box-emotion/index.js

```diff
@@ -0,0 +1,26 @@
+
+import React from 'react';
+import { ThemeProvider as EmotionTheme } from 'emotion-theming';
+import { styleFunction } from '@material-ui/core/Box';
+
+const materialSystemTheme = createMuiTheme();
+const BoxEmotion = styledEmotion('div')(styleFunction);
+
+const App = () => {
+  return (
+    <EmotionTheme theme={materialSystemTheme}>
+      <BoxEmotion
+        color="primary.main"
+        bgcolor="background.paper"
+        fontFamily="h6.fontFamily"
+        fontSize={['h6.fontSize', 'h4.fontSize', 'h3.fontSize']}
+        p={[2, 3, 4]}
+        fuu={Math.round(Math.random() * 10000)}
```

Comment:
Is this to defeat memoization? Could we use a more descriptive name?

Reply:
Didn't even noticed this, I just copied the examples from `material-ui-benchmark`. I would guess as you that it is to break the memoization. @oliviertassinari are you aware of this?

Reply:
I don't remember why it's here. Happy to drop it.